src/SFA.DAS.AssessorService.Web/Validators/Standard/AddStandardSearchViewModelValidator.cs

```diff
@@ -3,12 +3,12 @@
 
 namespace SFA.DAS.AssessorService.Web.Validators.Standard
 {
-    public class AddStandardViewModelValidator : AbstractValidator<AddStandardSearchViewModel>
+    public class AddStandardSearchViewModelValidator : AbstractValidator<AddStandardSearchViewModel>
     {
-        public AddStandardViewModelValidator()
+        public AddStandardSearchViewModelValidator()
         {
             const string searchMessage = "Enter 3 or more characters to search";
-            RuleFor(vm => vm.StandardToFind)
+            RuleFor(vm => vm.Search)
                 .NotEmpty().WithMessage(searchMessage)
```

Comment:
Think this check should happen after a `.Trim()` to account for whitespace

Reply:
Done that in both cases